recipes/pyeasyfuse/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "pyeasyfuse" %}
+{% set version = "2.0.3" %}
+{% set sha256 = "10bc6159d2da2f21f5e6a2bd3a04f9d1bc320e5a8259ce72f95b3cd3de58482d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyeasyfuse-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
```

Comment:
I think this should be `noarch:generic` as you have R dependencies.

Reply:
Good catch, I have changed noarch:python to generic in my next commit.